Source/PlasticSourceControl/Private/PlasticSourceControlOperations.cpp

```diff
@@ -171,8 +172,9 @@ bool FPlasticConnectWorker::Execute(FPlasticSourceControlCommand& InCommand)
 				// Now update the status of assets in the Content directory
 				// but only on real (re-)connection (but not each time Login() is called by Rename or Fixup Redirector command to check connection)
 				// and only if enabled in the settings
-				if (!GetProvider().IsAvailable() && GetProvider().AccessSettings().GetUpdateStatusAtStartup())
+				if (!PlasticSourceControlShell::GetShellIsWarmedUp() && GetProvider().AccessSettings().GetUpdateStatusAtStartup())
 				{
+					PlasticSourceControlShell::SetShellIsWarmedUp();
 					TArray<FString> ContentDir;
 					ContentDir.Add(FPaths::ConvertRelativePathToFull(FPaths::ProjectContentDir()));
 					PlasticSourceControlUtils::RunUpdateStatus(ContentDir, PlasticSourceControlUtils::EStatusSearchType::ControlledOnly, false, InCommand.ErrorMessages, States, InCommand.ChangesetNumber, InCommand.BranchName);
```

Comment:
As discussed earlier, we should consider running in on "EStatusSearchType::All" instead of ControlledOnly in order to warm the cache: the expected benefit is to

Reply:
I applied the discussed change in https://github.com/PlasticSCM/UEPlasticPlugin/pull/96/commits/24ee887c5ead1ee658bb55c68929e1edcaf5f2a9

This totally make sense so the feature is useful to users. Thanks!